### PR TITLE
Added a UC Contributing Guide (Issue #9)

### PR DESCRIPTION
**Pull request by vinijaiswal**
**PR Checklist**

- [*] A description of the changes is added to the description of this PR.
- [*] If there is a related issue, make sure it is linked to this PR.
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added or modified a feature, documentation in `docs` is updated

**Description of changes**
Added a Contribution Guide for the project. This PR also resolves an open issue #9.

It adds a file named **CONTRIBUTING.md** under the main project repository "unitycatalog". This serves as a guidance for interested community members to add their contributions to the project.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+- The GitHub review process for major features is also important so that organizations with commit access can come to agreement on design.
+- If it is appropriate to write a design document, the document must be hosted either in the GitHub tracking issue, or linked to from the issue and hosted in a world-readable location. 
+- Specifically, if the goal is to add a new extension, please read the extension policy.
+- Small patches and bug fixes don't need prior communication. If you have identified a bug and have ways to solve it, please create an [issue](https://github.com/unitycatalog/unitycatalog/issues) or create a [pull request](https://github.com/unitycatalog/unitycatalog/pulls).
+
+
+# Coding style
+We generally follow the [Apache Spark Scala Style Guide](https://spark.apache.org/contributing.html).
+
+# Sign your work
```
Comment (tdas) [inline CONTRIBUTING.md:25]: Let's add this later when we have to on board the Linux Foundation steps and start enforcing the DCO.

**Comment by tdas** [comment]
@vinijaiswal if you can address my comment, I can merge it.

File: CONTRIBUTING.md (hunk 1 above)
Comment (vinijaiswal) [reply CONTRIBUTING.md:25]: @tdas Thanks! You mean remove all the lines after 25 (Sign your work)?

**Comment by vinijaiswal** [comment]
@tdas 
I removed the DCO part from the initial commit. Please check if this is what you expected 🙌🏼

**Review by tdas — APPROVED** [review]

**Comment by tdas** [comment]
Thank you @vinijaiswal

**Pull request merged**